test/runtests.jl

```diff
@@ -12,7 +12,7 @@ const TvTYPES = [Float32, Float64, BigFloat]
 
 # Check That Tulip.version() matches what's in the Project.toml
 tlp_ver = Tulip.version()
-toml_ver = TOML.parsefile("../Project.toml")["version"]
+toml_ver = TOML.parsefile(joinpath(@__DIR__, "Project.toml"))["version"]
```

Comment:
this is an unrelated fix; `..` is relative to the working directory, while `@__DIR__` is the directory of the file itself, so I think this should be more reliable (e.g. it let me do `include("test/runtest.jl")` without having to switch to the test directory). Let me know if this is undesired and I can undo it.